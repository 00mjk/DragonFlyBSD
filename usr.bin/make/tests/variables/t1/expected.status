$DragonFly: src/usr.bin/make/tests/variables/t1/expected.status,v 1.1 2005/02/25 08:53:14 okumoto Exp $
0
