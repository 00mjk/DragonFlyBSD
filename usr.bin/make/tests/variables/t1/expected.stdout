$DragonFly: src/usr.bin/make/tests/variables/t1/expected.stdout,v 1.1 2005/02/25 08:53:14 okumoto Exp $
echo 0
0
echo 1
1
echo 0
0
# The following are soo broken why no syntax error?
echo 

echo 

echo 

echo 

echo 

echo 

echo 

echo 

echo 

echo 

echo 

echo 

